### ISSUE TITLE
Forge of Heroes' second ability cannot be activated when commander is a planeswalker.

### ISSUE THREAD

**Issue by tamaroth** [magefree/mage#5247]
When your commander is a planeswalker, Forge of Heroes' ability cannot be activated. 

This is because ``CommanderPredicate()`` checks only for creatures and will always return false when a commander is not a creature. 

Currently, the predicate is as follows:

```
return input.isCreature()
       && owner != null
       && owner.getCommandersIds().contains(input.getId());
```

If we use 

```
return (input.isCreature() || input.isPlaneswalker())
       && owner != null
       && owner.getCommandersIds().contains(input.getId());
```
Everything should work as intended.

**Comment by tamaroth** [comment]
I will correct the code and submit the PR.

**Comment by emerald000** [comment]
Your commander can also be neither of those. If it's been enchanted by a Imprisoned in the Moon, it's a Land. There shouldn't be any check about card type.

**Comment by JayDi85** [comment]
Fixed by #5248
